layouts/index.html

```diff
@@ -28,7 +28,7 @@
         	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="{{ T "input_placeholder_email_address" }}" aria-label="email" required>
             <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
             <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_3885586f8f1175194017967d6_11c1b8bcb2" tabindex="-1" value=""></div>
-            <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+            <div class="clear"><input type="submit" value="{{ T "subscribe_button" }}" name="{{ T "subscribe_button" }}" id="mc-embedded-subscribe" class="button"></div>
```

Comment:
What is the point of localizing the `name` attribute?

Reply:
corrected